packages/plugin-mapbox/src/test/java/com/openmobilehub/android/maps/plugin/mapbox/presentation/maps/OmhMarkerImplTest.kt

```diff
@@ -360,7 +360,7 @@ class OmhMarkerImplTest {
             // marker cleanup
             style.removeStyleLayer(OmhMarkerImpl.getSymbolLayerID(markerUUID))
             style.removeStyleSource(OmhMarkerImpl.getGeoJsonSourceID(markerUUID))
-            style.removeStyleImage(omhMarker.getMarkerIconID(omhMarker.isCustomIconSet))
+            style.removeStyleImage(omhMarker.lastMarkerIconID!!)
```

Comment:
please do not use !! , could you change it to safe call?

Reply:
Refactored in 9327fcf6737a9be81ef21be39617c736ec344964